keps/sig-apps/2214-indexed-job/README.md

```diff
@@ -206,16 +207,28 @@ type JobSpec struct {
   // `NonIndexed`.
   CompletionMode CompletionMode
 }	
+
+type JobStatus struct {
+  ...
+
+  // CompletedIndexes holds the completed indexes when .spec.completionMode =
+  // "Indexed" in a text format. The indexes are represented as decimal integers
+  // separated by commas. The numbers are listed in increasing order. Two or
+  // more consecutive numbers are compressed and represented by the first and
+  // last element of the series, separated by a hyphen.
+  // For example, if the completed indexes are 1, 3, 4, 5 and 7, they are
+  // represented as "1,3-5,7".
+  CompletedIndexes string
```

Comment:
(sorry, I know this was a big discussion about this in the main KEP PR, but I just thought of this)

is this Job status field the only way to retrieve this information? If not, do we need precise resolution in this string? if approximate CompletedIndexes is okay, we could bucket large lists and report the number of completed indexes in each bucket (like `1-1000:727/1000,1001-2000:1000/1000,...`). much smaller string.

Reply:
you can remove the `/1000` for even shorter strings, but I think that helps with readability

Reply:
This information doesn't give us anything, because we still don't know which of those 1-1000 should be restarted.

Reply:
Ah I misread the KEP. I thought this was just for kubectl, but it turns out the controller uses this information as the source of truth.

(github won't let me resolve this comment, but consider this resolved)